zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/ObjectenApiPlugin.kt

```diff
@@ -77,6 +78,20 @@ class ObjectenApiPlugin(
         }
     }
 
+    fun getObjectAtIndex(objectUrl: URI, index: Int): ObjectRecord {
+        withLoggingContext("objectUrl" to objectUrl.toString()) {
+            logger.debug { "Getting Objecten API object with url '$objectUrl' and index '$index'" }
+
+            val objectUrlWithIndex = UriComponentsBuilder
+                .fromUri(objectUrl)
+                .pathSegment(index.toString())
+                .build()
+                .toUri()
```

Comment:
Could this code be moved to the client?

Reply:
Not sure i agree, since this is not just building the URL to do a call, but also because that is the identifier of the object.

Reply:
The index is not part of the identifier of the object. 

Ideally, anything related to a web request belongs in the client.

Reply:
I'll take a look

Reply:
Fixed.